src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -383,25 +381,17 @@ private void ProcessTraceroute(string targetNameOrAddress)
 #endif
                 var hopAddressString = discoveryReply.Address.ToString();
 
+                if (!InitProcessPing(hopAddressString, out string routerName, out _))
```

Comment:
Not really part of this PR, but `InitProcessPing()` should probably be `ResolveToNameOrAddress()` or something more descriptive

Reply:
Yeah, I agree, for sure. I can change it here if you want. 🙂

Reply:
@vexx32 if you have a separate commit for it, would make it easier to review and wouldn't need a new PR